ci/generate_version_tag.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+
+import semver
+
+
+def main(changes, old_tag, prerelease_tag, major_release):
```

Comment:
Could we add typehints and a usual docstring that describes each parameter?

Reply:
Done.